src/app/components/RecipeCards/CardStarred/index.tsx

```diff
@@ -0,0 +1,64 @@
+import './index.css';
+import { Card, CardContent, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import React from 'react';
+
+export type StepItemType = {
+  id: number;
+  title: string;
+  description: string;
+};
+
+type Props = {
+  item: StepItemType;
+};
+
+export default (props: Props): JSX.Element => {
```

Comment:
Isn't a 'function' and a name for the function missing? I probably works just different and I haven't seen this method before? 🤔

Reply:
it's an anonymous function :) it just doesn't have a name, works the same as a named function. 
According to my knowledge, it makes sense to use them when you don't reuse the function, or use it only a couple of times. I haven't had any problems with them yet :D
oh, and the missing 'function' part is solved by using '( ) =>' :D 

Thank you for the review!